provider/pdns/pdns.go

```diff
@@ -58,8 +59,17 @@ const (
 	retryLimit = 3
 	// time in milliseconds
 	retryAfterTime = 250 * time.Millisecond
+	// record types which require to have trailing dot
```

Comment:
not sure why this comment is here

Reply:
Good catch. I should have been few lines bellow. Fixed in commit https://github.com/kubernetes-sigs/external-dns/pull/5992/commits/c496fd94d37cd3885e701b7ca22c4b77ee396263